src/routes/friends/requests.tsx

```
@@ -0,0 +1,71 @@
+        ]}
+        sectionSelected={sectionSelected}
+        onSectionSelect={setSectionSelected}
+      />
+      <div>
+        {sectionSelected === "1"
+          ? friendInvitations?.pages[0]?.data.map((request) => (
+              <FriendRequest
+                user_id={request.user_id_requested}
+                status={request.status}
+                type="received"
+              />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)